sdk/js/src/service/gateways.js

```diff
@@ -184,6 +186,33 @@ class Gateways {
     // Combine all stream sources to one subscription generator.
     return combineStreams(streams)
   }
+
+  // Gateway Configuration Server
+
+  async getGlobalConf(gatewayId, baseURL, token) {
+
+    let Authorization = null
+    if (typeof token === 'string') {
+      Authorization = `Bearer ${token}`
+    }
+    else if (typeof token === 'function') {
+      const tkn = (await token()).access_token
+      Authorization = `Bearer ${tkn}`
+    }
+
+    let acc = axios.create({
+      baseURL: baseURL,
+      headers: {
+        Authorization
+      },
+    })
+
+    let url = `${baseURL}/gcs/gateways/${gatewayId}/semtechudp/global_conf.json`
+
+    const response = await acc.get(url)
```

Comment:
```suggestion
    const response = await axios.get(url, {
      baseURL: baseURL,
      headers: {
        Authorization
      },
    })
```

Haven't checked the axios API, but you can do something like this instead of using the instance.